### PR TITLE
Add metal3-io/community repository in prow config

### PR DESCRIPTION
**Pull request by kashifest**
This PR also adds a markdownlint prow job for the repo.

### CODE REVIEW

File: prow/manifests/overlays/metal3/config.yaml [hunk 1]
```diff
@@ -899,6 +900,37 @@ presubmits:
             image: docker.io/golang:1.19
             imagePullPolicy: Always
 
+  metal3-io/community:
+    - name: markdownlint
+      run_if_changed: '\.md$'
```
[the remark this answers is not in the document]
Comment (kashifest) [reply prow/manifests/overlays/metal3/config.yaml]: Done

**Comment by smoshiur1237** [comment]
/override test-ubuntu-integration-main
 These changes are not tested by integration test.

**Comment by kashifest** [comment]
> We should first add OWNERS etc befiore activating Prow?

I have that PR up as well, now thinking which one should land first

**Comment by tuminoid** [comment]
> > We should first add OWNERS etc befiore activating Prow?
> 
> I have that PR up as well, now thinking which one should land first

If you don't have OWNERS in place, landing this first making merging anything else more difficult. You basically need to override Prow and manually merge stuff etc. So its just less steps to merge the OWNERS PR first IMO.

**Comment by kashifest** [comment]
ok I will fetch out OWNERS file and merge it first

File: prow/manifests/overlays/metal3/config.yaml [hunk 2]
```diff
@@ -899,6 +900,37 @@ presubmits:
...
+          - args:
+              - ./hack/shellcheck.sh
+            command:
+              - sh
+            env:
+              - name: IS_CONTAINER
+                value: "TRUE"
+            image: docker.io/koalaman/shellcheck-alpine:v0.9.0@sha256:e19ed93c22423970d56568e171b4512c9244fc75dd9114045016b4a0073ac4b7
+            imagePullPolicy: Always
+
```
Comment (tuminoid) [inline prow/manifests/overlays/metal3/config.yaml:932]: Double empty line.

File: prow/manifests/overlays/metal3/config.yaml [hunk 3]
```diff
@@ -899,6 +900,37 @@ presubmits:
...
+              - ./hack/markdownlint.sh
+            command:
+              - sh
+            env:
+              - name: IS_CONTAINER
+                value: "TRUE"
+            image: docker.io/pipelinecomponents/markdownlint:0.13.0@sha256:9c0cdfb64fd3f1d3bdc5181629b39c2e43b6a52fc9fdc146611e1860845bbae0
+            imagePullPolicy: Always
+    - name: shellcheck
+      run_if_changed: '((\.sh)|^Makefile)$'
```
Comment (tuminoid) [inline prow/manifests/overlays/metal3/config.yaml:919]: :+1: 
Thanks for picking it up.

**Review by tuminoid — COMMENTED** [review]
/approve
/hold

One nit, plus OWNERS PR should land first.

**Review by Rozzii — APPROVED** [review]
/lgtm
/test-ubuntu-integration-main

File: prow/manifests/overlays/metal3/config.yaml (hunk 2 above)
Comment (smoshiur1237) [reply prow/manifests/overlays/metal3/config.yaml:932]: Should we fix the double empty line?

**Comment by smoshiur1237** [comment]
/hold
We can skip the test after removing the empty line.

**Pull request merged**